.gitpod.yml

```diff
@@ -0,0 +1,3 @@
+tasks:
```

Comment:
We can remove this file as #1005 got rejected.

Reply:
got it.. so we will remove gitpod support.